CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ that users understand how the changes affect the new version.
 
 version develop
 ---------------------------
++ Make YAML file with docker images optional.
```

Comment:
It isn't really optional now right? You are just setting a default.

Reply:
In theory, you could use a tagged version of the `subreads-processing` pipeline, instead of the current develop. That way the images are also stable for that pipeline.
I think we would have to release the pipeline for that first. It should be ready for that.